### ISSUE TITLE
Port ACS to Java 8

### ISSUE THREAD

**Issue by moraklein** [ACS-Community/ACS#30]
Make ACS compatible with the latest release of the JDK, Java 8.
New features in Java, such as lambda functions, as well as new functionality for GUIs in JavaFX are part of this release. Oracle has announced that it will soon stop issuing updates for the Java 7 SDK. ACS has been tested to works without problems with the latest JDK 8 update, but building ACS fails on the compilation of JacORB 3.4; the problem has been fixed in JacORB 3.5, and merging that version into ACS might be all that is required.

**Comment by moraklein** [comment]
Note that Java 8 support will be provided by ALMA through ACS release 2015.4.

**Comment by acaproni** [comment]
I guess this ticket can be closed as ESO already provided this enhancement in 2015.4: https://ictwiki.alma.cl/twiki/bin/view/Main/Acs2015-4ReleaeNotes

**Comment by jhschwarz** [comment]
I don't think that we can close this until version 2015.4 of ACS has been ported to the Community (i.e., to here). AFAIK we're still at 2015.2.

**Comment by acaproni** [comment]
Sure Joe it wil be available merging 2015.4 however there is no port to be done but a merge
